sheoranharis.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Harish Sheoran",
+    "type": "personal",
+    "bio": "Pythonista",
+    "avatar": "https://github.com/sheoranharis.png",
+    "links": [
+        {
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/sheoranharis",
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/sheoranharis",
+            "icon": "twitter"
+        }
+      {
```

Comment:
```suggestion
        },
        {
```

Reply:
This should be fixed automatically by the CI. 🙂